package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-skyscanner",
-  "version": "2.0.0",
+  "version": "3.0.0",
```

Comment:
How are you planning to publish this? Usually i'd use `np` (installed as a global npm module) to publish it. This allows you too choose which version you want and it updates the `package.json` for you.

Perhaps we should add publishing docs to the readme for this

Reply:
Ah ok - I wondered what `np` was for!